app/views/admin/locations/_form.html.erb

```diff
@@ -121,6 +121,17 @@
   <%= f.label "hidden_true_#{location.id}", 'No', class: 'l-location-status__label' %>
 </div>
 
+<% if policy(location).admin? %>
+  <div class="form-group <%= 'form-group--error' if f.object.errors[:online_booking_enabled].any? %>" id="online_booking_enabled">
+    <%= f.label :online_booking_enabled, 'Online bookings?', class: 'block' %><br/>
+    <%= render 'error', error_messages: location.errors[:online_booking_enabled] %>
+    <%= f.radio_button :online_booking_enabled, true, class: 'l-location-status__checkbox', id: "location_online_booking_enabled_true_#{location.id}" %>
```

Comment:
Could this not be simplified with a checkbox? The backing attribute defaults to `false` so it should just work ™️.